vue_chat_front/src/components/Header.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <header class="bg-grey-400 py-4">
+    <div class="container m-auto flex flex-wrap items-center justify-end">
+      <div class="flex-1 flex items-center">
+        <a href="/" class="uppercase text-sm font-mono pl-4 font-semibold no-underline text-indigo-500 hover:text-indigo-400">Chat App</a>
+      </div>
+      <div>
+        <router-link to="/" class="text-gray-500 px-2 no-underline" v-if="!signedIn()">Sign in</router-link>
+        <router-link to="/signup" class="text-gray-500 px-2 no-underline" v-if="!signedIn()">Sign Up</router-link>
```

Comment:
RouterLinkを使う際は直接パスを記述するのは控えたほうが良いです。
`router/index.js`に記載しているnameを指定するようにしましょう。
直接パスを記述してしまうと、後々パスを変更する事になった際に全ての箇所を修正する必要が出てきますが、
nameを使う事により`router/index.js`の修正だけで済むようになります。
```suggestion
        <router-link :to="{ name: 'Signup' }" class="text-gray-500 px-2 no-underline" v-if="!signedIn()">Sign Up</router-link>
```